### DISCUSSION TITLE
Your account has been flagged.

### DISCUSSION THREAD

**Discussion by RajaSabiqISX** [community/community#82771] [New to GitHub]
I just created my new GitHub account on my company mail which I get from my shared hosting and just when I created the account it showed me this notification can anyone help me in this matter

**Comment by queenofcorgis** [comment]
Hi @RajaSabiqISX ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
